src/index.ts

```diff
@@ -10,7 +11,9 @@ export class Aracari<T extends HTMLElement = HTMLElement> {
   constructor(root: T, options: Partial<Config> | undefined = {}) {
     this.root = root;
     this.config = {
-      textNodeType: options.textNodeType || Node.TEXT_NODE
+      textNodeType: options.textNodeType || Node.TEXT_NODE,
+      createTextNode:
+        options.createTextNode || document.createTextNode.bind(document)
```

Comment:
I was thinking maybe I did not need this, but I do need the binding.

<img width="298" alt="Screen Shot 2020-10-26 at 9 51 54 PM" src="https://user-images.githubusercontent.com/578259/97258416-9b8b4280-17d5-11eb-9e3a-308a1b56e507.png">